### ISSUE TITLE
Broken or outdated links in Understanding Success Criterion 2.4.4: Link Purpose (In Context)

### ISSUE THREAD

**Issue by cstrobbe** [w3c/wcag#2773]
[Understanding Success Criterion 2.4.4: Link Purpose (In Context)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html) for WCAG 2.1 and [Understanding Success Criterion 2.4.4: Link Purpose (In Context)](https://www.w3.org/WAI/WCAG22/Understanding/link-purpose-in-context.html) for WCAG 2.2 lists three links under "Related Resources".

* The first one, [[Using Link Titles to Help Users Predict Where They Are Going](https://www.nngroup.com/articles/using-link-titles-to-help-users-predict-where-they-are-going/)](https://www.nngroup.com/articles/using-link-titles-to-help-users-predict-where-they-are-going/), redirects to [Using the Title Attribute to Help Users Predict Where They Are Going](https://www.nngroup.com/articles/title-attribute/). Since the `title` attribute cannot be relied upon to be read by screen readers, I would suggest replacing this link. One alternative is [Writing Hyperlinks: Salient, Descriptive, Start with Keyword](https://www.nngroup.com/articles/writing-links/) on the same site (article dated March 2014). There's also [Making Accessible Links: 15 Golden Rules For Developers](https://www.sitepoint.com/15-rules-making-accessible-links/) by Gian Wild (a former WCAG WG participant) from 2014.
* The link [Hidden barriers - out of sight](http://www.rnib.org.uk/blogs/expert-series?Name=Hidden%20barriers) no longer works and an article of the same title can no longer be found elsewhere on the RNIB website. In my experience, links on the RNIB website get outdated too quickly for inclusion in Understanding documents.

(I don't know whether links in Understanding WCAG 2.1 still get fixed; that document is simply the version where I first noticed the issue.)

**Comment by giacomo-petri** [comment]
**Official Working Group Response**

> The first one, [Using Link Titles to Help Users Predict Where They Are Going](https://www.nngroup.com/articles/using-link-titles-to-help-users-predict-where-they-are-going/), redirects to [Using the Title Attribute to Help Users Predict Where They Are Going](https://www.nngroup.com/articles/title-attribute/). Since the title attribute cannot be relied upon to be read by screen readers, I would suggest replacing this link. One alternative is [Writing Hyperlinks: Salient, Descriptive, Start with Keyword](https://www.nngroup.com/articles/writing-links/) on the same site (article dated March 2014). There's also [Making Accessible Links: 15 Golden Rules For Developers](https://www.sitepoint.com/15-rules-making-accessible-links/) by Gian Wild (a former WCAG WG participant) from 2014.

My understanding is that the intent is to clarify that title should provide secondary information (not essential for understanding the link itself), since it's used as a description when an accessible name is present. This makes it a "related resource" explaining how to use the attribute appropriately, not for naming.

> The link [Hidden barriers - out of sight](http://www.rnib.org.uk/blogs/expert-series?Name=Hidden%20barriers) no longer works and an article of the same title can no longer be found elsewhere on the RNIB website. In my experience, links on the RNIB website get outdated too quickly for inclusion in Understanding documents.

This link has been already removed.

**Comment by patrickhlauke** [comment]
do we want to edit the first link to point to its new location after the redirect?

**Comment by mbgower** [comment]
Completed a working group review with general support so changed the draft response to the official response.
Closing the issue. Feel free to reopen if you have additional information/perspective to provide.
